### PR TITLE
Added Bangla Tutorial for LOJ-1012 Guilty Prince 

### PR DESCRIPTION
**Pull request by walleeva2018**

### CODE REVIEW

File: 1012/bn.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                A[i][j]=s[j];
+                if(A[i][j]=='@')
+                {
+                    x=i;
+                    y=j;
+                }
+            }
+        }
+```
+
```
Comment (rebornplusplus) [inline 1012/bn.md:50]: Implementation plans.

File: 1012/bn.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    cnt++;
+    vis[x][y]=1;
+    dfs(x+1,y);
+    dfs(x,y+1);
+    dfs(x-1,y);
+    dfs(x,y-1);
+}
+```
+### কিছু সতর্কতা
+প্রতি কেস এর পর vis এবং cnt ক্লিয়ার করতে হবে
```
Comment (rebornplusplus) [inline 1012/bn.md:79]: Implementation plans.

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
Hello @walleeva2018. Great efforts. The picture is really nice!

However, I feel that the tutorial consists _mostly_ about implementation plans i.e. how you would write the code to solve it. It's better if you lose them and re-write to how you summarize the solution without the codes and all; in plain sentences.

I see that there is already an existing tutorial in English (#108) and it's a good one. So what you could also do instead, is translate that one.

File: 1012/bn.md (hunk 2 above)
Comment (walleeva2018) [reply 1012/bn.md:79]: Okay Sir , I am fixing it

Comment (walleeva2018) [reply 1012/bn.md]: I tried to fix it Sir. Can you review it and tell me what to do next

**Review by rebornplusplus — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**